deploy/development.sh

```diff
@@ -8,8 +8,6 @@ kubectl delete deployments.apps localstack -n jvm-build-service
 DIR=`dirname $0`
 kubectl apply -f $DIR/namespace.yaml
 kubectl config set-context --current --namespace=jvm-build-service
-find $DIR -name maven-v0.2.yaml -exec sed -i s/redhat-appstudio/${QUAY_USERNAME}/ {} \;
```

Comment:
@gabemontero why did you add this? It should already be replaced by an overlay. This is modifying the version controlled files, which means it is likely the changes will be accidentally commited.

Reply:
yeah this was meant to be a placeholder while I vetted things today ... I just didn't put comments to that effect before I signed off last night